packages/backend/src/queryBuilder.ts

```diff
@@ -915,7 +919,12 @@ export const buildQuery = ({
             sortedDimension &&
             sortedDimension.timeInterval === TimeFrames.MONTH_NAME
         ) {
-            return sortMonthName(sortedDimension);
+            shouldWrapQueryCTE = true;
```

Comment:
Like the week name, this sql also needs to be wrapped in CTE https://github.com/lightdash/lightdash/pull/10806/files#diff-641ce89defd0e168c1fcf517d68dbfaecbcdd76f6063158efae1d2826a086ac1R933